pysal/weights/Wsets.py

```diff
@@ -531,3 +531,326 @@ def w_clip(w1, w2, outSP=True, silent_island_warning=False):
     return wc
 
 
+def w_stack(ws, silent_island_warning=False):
```

Comment:
No clue if it's used anywhere else in the library, but I used this pattern in `by_col_array`:
```python
filehandler.by_col_array('HOVAL', 'INC', 'CRIME')
# same as 
filehandler.by_col_array(['HOVAL', 'INC', 'CRIME']) 
```